src/command.c

```diff
@@ -623,6 +623,8 @@ static int process_password(int p, char *password)
 	  pprintf(p,"\n  ** LOGGED IN AS HEAD ADMIN **\n");
 	  pp->adminLevel = ADMIN_GOD;
   }
+  
+  pprintf(p,"\n**** Starting FICS session as %s%s ****\n", pp->name, CheckPFlag(p, PFLAG_REG) ? "": "(U)");
```

Comment:
Sorry to nitpick but in hindsight I guess "ICS" or simply "Starting session" is probably better, since most people understand FICS to mean freechess.org.